src/libraries/System.Text.RegularExpressions/tests/RegexReductionTests.cs

```diff
@@ -456,7 +468,7 @@ public void PatternsReduceIdentically(string pattern1, string pattern2)
         [InlineData("(?i:abcd)|abcd", "abcd|abcd")]
         [InlineData("abcd|(?i:abcd)", "abcd|abcd")]
         // Not applying auto-atomicity
```

Comment:
does this comment apply to the lines below? as it appears that it is applying auto-atomicity.

Reply:
worth keeping the `[InlineData("a*b*", "(?>a*)b*")]` case? I guess I'm not inclined to reason too much about whether something is necessary to add more coverage or not so long as it's not self evidently equivalent to another..

Reply:
oh disregard that, I see it above.

Reply:
> as it appears that it is applying auto-atomicity.

This test validates they're _not_ the same.

Reply:
Oops I looked above at the method name, but of course it's below for [InlineData], I did that before..